src/components/operations/operation-details/ko/runtime/operation-console.ts

```diff
@@ -749,7 +748,27 @@ export class OperationConsole {
     }
 
     public toggleRequestSummarySecrets(): void {
+        if (!this.secretsRevealed()) {
+            this.consoleOperation().request.meaningfulHeaders().forEach(header => header.revealed(true));
```

Comment:
`this.consoleOperation().request.meaningfulHeaders().forEach(header => header.revealed(true))`
`this.consoleOperation().request.meaningfulHeaders().forEach(header => header.revealed(false))`

Maybe just this:
`this.consoleOperation().request.meaningfulHeaders().forEach(header => header.revealed(this.secretsRevealed()));`